can_tools/scrapers/official/TN/tn_state.py

```diff
@@ -0,0 +1,208 @@
+import pandas as pd
+import requests
+import us
+
+from can_tools.scrapers.base import CMU
+from can_tools.scrapers.official.base import CountyDashboard
+from can_tools.scrapers.official.base import StateDashboard
+
+
+class TennesseeCasesDeathsTests(CMU, StateDashboard):
```

Comment:
I think I would vote that we name this `TennesseeState` and the other class `TennesseeCounty`

Happy to hear alternative viewpoints

Reply:
Sounds great, I was thinking about that after I had submitted the pull request.  I'll split them up.  Should we split the `tn_state.py` file into multiple files as well?

Reply:
Thanks for the thoughts here. 

I'm excited to see all the data come in!

I think that for each of the datasets in your list above, the second class name listed makes sense to me.

Are you ok with those names?

Reply:
Yes, I'll get the classes renamed now.